hypothesis-python/src/hypothesis/_strategies.py

```diff
@@ -1305,6 +1305,25 @@ def from_type(thing):
        other elements in the lookup.
     4. Finally, if ``thing`` has type annotations for all required arguments,
        it is resolved via :func:`~hypothesis.strategies.builds`.
+
+    There is a valuable recipe for leveraging ``from_type`` to generate
+    "everything except" values from a specified type. I.e.
+
+    .. code-block:: pycon
+
+        >>> def everything_except(excluded_types):
+        ...     return (
+        ...         # draws from the list of registered builtin types
```

Comment:
Given the prose description below, I think we can ditch the comments here and use `python` instead of `pycon` syntax highlighting (thus ditching the `>>>` and `...` as well).  I'd even compress it a little more:

```python
def everything_except(excluded_types):
    return (
        st.from_type(type).flatmap(st.from_type)
        .filter(lambda x: not isinstance(x, excluded_types))
    )